tests/test_xdsl_op_correctness.py

```diff
@@ -43,7 +41,7 @@ def test_u_plus1_conversion():
     assert np.isclose(norm1, norm2, atol=1e-5, rtol=0)
     assert np.isclose(norm1, 23.43075, atol=1e-5, rtol=0)
 
-@pytest.mark.xfail(reason="Deprecated, will be dropped")
+@pytest.mark.xfail(reason="Needs a fix in offsets")
```

Comment:
Where is the fix needed and what's wrong?

Reply:
The: 
```
https://github.com/search?q=repo%3Axdslproject%2Fdevito%20time_offset_to_field&type=code
```

is not including the specific offset that is needed for this case. Need to look for some way to cover this case. SHould not be difficult